packages/api-catalog-spa/src/pages/ApiCUDPage/components/APIReview/APIReview.tsx

```diff
@@ -0,0 +1,193 @@
+import { useState } from 'react';
+import { useFormContext } from 'react-hook-form';
+import {
+  Card,
+  CardBody,
+  CardTitle,
+  Checkbox,
+  DataList,
+  DataListCell,
+  DataListItem,
+  DataListItemCells,
+  DataListItemRow,
+  Grid,
+  GridItem,
+  List,
+  ListItem,
+  Split,
+  SplitItem,
+  Stack,
+  StackItem,
+  Title,
+} from '@patternfly/react-core';
+
+import { config } from 'config';
+import { CatalogBigButton, ReadMore } from 'components';
+import { FormData } from 'pages/APICudPage/APICudPage.types';
+
+import { Description } from './components/Description';
+
+const apiOptions: Record<string, { title: string; desc: string; image: string }> = {
+  REST: {
+    title: 'REST',
+    desc: 'Confirms to OpenAPI specs',
```

Comment:
```suggestion
    desc: 'API that conforms to the constraints of Representational state transfer architecture,
```

Reply:
This was already discussed before. We took a short sentence like that to keep it consistent with the graphql one and also to keep it without breaking the design.